nixos/modules/services/misc/prowlarr.nix

```diff
@@ -29,7 +31,7 @@ in
         Type = "simple";
         DynamicUser = true;
         StateDirectory = "prowlarr";
-        ExecStart = "${pkgs.prowlarr}/bin/Prowlarr -nobrowser -data=/var/lib/prowlarr";
+        ExecStart = "${cfg.package}/bin/Prowlarr -nobrowser -data=/var/lib/prowlarr";
```

Comment:
nit: this could use lib.getExe

Reply:
No, it could not.  meta.mainProgram needs to be set for getExe to work.

Reply:
Might as well do that then, PR incoming.

Reply:
See #243479.